comprehensive/lms/templates/navigation.html

```diff
@@ -220,7 +220,9 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
 <%include file="help_modal.html"/>
 ## Don't show cookie-banner if already accepted.
 <script type="text/javascript">
+
     if (cookieNotice.getCookie("cookie-banner") == "true") {
```

Comment:
nit: i generally prefer triple quality === , but i'm not blocking on this
https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons